AxonDeepSeg/visualization/visualize.py

```diff
@@ -97,10 +97,11 @@ def _create_fig_helper(overlayed_img, fig_title):
         ax.imshow(overlayed_img, cmap="hsv", alpha=0.7)
         return fig
 
-    path_img = path + "/image.png"
+    path_img = path / "image.png"
     mask = False
+    listdir = [item.name for item in path]
```

Comment:
`listdir` is not a good variable name, as there exists `os.listdir`, which is sometimes imported as `from os import listdir`, causing a conflict.